Bio/SeqIO/SffIO.py

```diff
@@ -1011,7 +1011,7 @@ def _sff_read_seq_record(self, stream):
         )
         # Dirty trick to speed up this line:
         # record.letter_annotations["phred_quality"] = quals
-        dict.__setitem__(record._per_letter_annotations, "phred_quality", quals)
+        dict.__setitem__(record.letter_annotations, "phred_quality", quals)
```

Comment:
This should also remove the "dirty trick" comment above. And is this change slowing it down - why not change to use ``record = SeqRecord._from_validated(...)`` as per the FASTQ changes?

Reply:
Yeah, switching to _from_validated would probably be better. This was an earlier change to mollify the type checker because _per_letter_annotations can now be None if it hasn’t be initialized yet